### PR TITLE
reenabled disabled tests

### PR DESCRIPTION
**Pull request by vlada-shubina**
Reenabled previously disabled unit tests
Added more tests for aspnetcore

### CODE REVIEW

**Review by DavidKarlas — APPROVED** [review]

**Review by AR-May — APPROVED** [review]

File: tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs [hunk 1]
```diff
@@ -98,7 +98,7 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
         [InlineData("2.2", "dotnet")]
         [InlineData("3.0", "dotnet")]
         [InlineData("3.1", "dotnet")]
-        // [InlineData("5.0", "dotnet")] - Broken
+        [InlineData("5.0", "dotnet")]
```
Comment (ladipro) [inline tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:101]: Shouldn't `"release/5.0", "dotnet"` work as well, now that 5.0 GA'ed?

File: tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs [hunk 2]
```diff
@@ -110,6 +110,10 @@ public void WhenRuntimeParametersArePassedToInstallScripts(string runtime, strin
         [InlineData("Current", "aspnetcore")]
         [InlineData("LTS", "aspnetcore")]
         [InlineData("master", "aspnetcore")]
+        [InlineData("2.2", "aspnetcore")]
+        [InlineData("3.0", "aspnetcore")]
+        [InlineData("3.1", "aspnetcore")]
+        [InlineData("5.0", "aspnetcore")]
```
Comment (ladipro) [inline tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:116]: Same question here: Are `"release/3.0", "aspnetcore"`, `"release/3.1", "aspnetcore"`, and `"release/5.0", "aspnetcore"` really still broken?

**Review by ladipro — APPROVED** [review]

File: tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs (hunk 1 above)
Comment (vlada-shubina) [reply tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:101]: no, it's still broken

File: tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs (hunk 2 above)
Comment (vlada-shubina) [reply tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:116]: They are still broken. The cause is they are not published to azure blob. 
`aspnetcore\Runtime\release` has only 2.1 and 2.2
`Runtime\release` has 1.0.0, 1.1.0, 2.0.0, 2.1, 2.2, 3.0, 3.1 and uwp6.0(?) 

@mmitche is it expected behavior? Are those release/XX channels are not expected anymore or is it a bug?
SDK still have all the versions.

Comment (mmitche) [reply tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:116]: Those should work. @rbhanda @wtgodbe

Comment (wtgodbe) [reply tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:116]: we publish to `aspnetcore\runtime\{BuildVersion}` - should we be copying things to folders based on major.minor? Would that be up to us, or would that be part of release staging?

Comment (vlada-shubina) [reply tests/Install-Scripts.Test/GivenThatIWantToInstallTheSdkFromAScript.cs:116]: Moved conversation to https://github.com/dotnet/install-scripts/issues/124

**Pull request merged**